pkg/blocks/step.go

```diff
@@ -22,241 +22,187 @@ package blocks
 import (
 	"errors"
 	"fmt"
-	"runtime"
-	"strings"
 
 	"github.com/facebookincubator/ttpforge/pkg/logging"
-	"github.com/facebookincubator/ttpforge/pkg/outputs"
-	"go.uber.org/zap"
 	"gopkg.in/yaml.v3"
 )
 
-// Constants representing supported executor types.
-const (
-	ExecutorPython     = "python3"
-	ExecutorBash       = "bash"
-	ExecutorSh         = "sh"
-	ExecutorPowershell = "powershell"
-	ExecutorRuby       = "ruby"
-	ExecutorBinary     = "binary"
-	ExecutorCmd        = "cmd.exe"
-)
-
-// StepType denotes the type of a step in a TTP.
-type StepType string
-
-// Constants for defining the types of steps available.
-const (
-	StepCreateFile = "createFileStep"
```

Comment:
step type strings are no longer required - one less thing to update when creating a new step